src/components/pages/OnboardPage.tsx

```diff
@@ -64,17 +60,28 @@ const OnboardPage = ({ unmountSelf }: IOnboardPageProps) => {
           </h1>
         </div>
         <div className="mt-10 flex flex-col">
-          <div className="flex flex-col gap-4" onClick={moveToMain}>
-            {mockPorts.map((port) => {
-              return (
+          <div className="flex flex-col gap-4">
+            {activePorts ? (
+              activePorts.map((port, index) => (
```

Comment:
There needs to be a way to refresh the page, maybe something taking the place of the "continue without radio" button" in the UI.

You can use something like in this:

https://upmostly.com/tutorials/how-to-refresh-a-page-or-component-in-react

Reply:
Added.